### PR TITLE
To corona ex

### PR DESCRIPTION
**Pull request by gammesB**
example https://to-corona-ex.com/comics/20000000051072

### CODE REVIEW

File: src/web/mjs/connectors/To-Corona-Ex.mjs [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+            };
+        });
+    }
+    async _getPages(chapter) {
+        let request = new Request(
+            "https://api.to-corona-ex.com/episodes/" + chapter.id + "/begin_reading",
+            this.requestOptions
+        );
+        const chapterinfo = await this.fetchJSON(request);
+        let drmhash = [];
```
Comment (27136773) [inline src/web/mjs/connectors/To-Corona-Ex.mjs]: May remove unused variable

File: src/web/mjs/connectors/To-Corona-Ex.mjs [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+        let imagelinks = [];
+        for (let i = 0; i < chapterinfo.pages.length; i++) {
+            drmhash.push(chapterinfo.pages[i].drm_hash);
+            imagelinks.push(this.createConnectorURI(chapterinfo.pages[i].page_image_url));
```
Comment (27136773) [inline src/web/mjs/connectors/To-Corona-Ex.mjs]: May use `.map` instead of loop

File: src/web/mjs/connectors/To-Corona-Ex.mjs [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+        }
+        return imagelinks;
+    }
+    _handleConnectorURI(payload) {
+        let promise;
+        switch (true) {
```
Comment (27136773) [inline src/web/mjs/connectors/To-Corona-Ex.mjs]: May use guard to fall thorugh
```javascript
if(!payload.includes('https://cdn.to-corona-ex.com/pages/images/')) {
    return super._handleConnectorURI(payload);
}
let data = await this.prepareForDescramble(payload);
data = this._blobToBuffer(data);
this._applyRealMime(data);
return data;
```

**Comment by 27136773** [comment]
The icon is to large, it should not exceed 16 KB and ideally hava a resolution of 64x64 px

**Review by 27136773 — APPROVED** [review]

**Comment by lasthm** [comment]
Is it possible to download monthly paid member chapters？

**Pull request merged**